ExpenseTracker.Core/Models/Transactions/Exceptions/AlreadyExistsTransactionException.cs

```diff
@@ -13,5 +13,9 @@ public class AlreadyExistsTransactionException : Xeption
         public AlreadyExistsTransactionException(Exception innerException)
             : base(message: "Transaction with the same id already exists.", innerException)
```

Comment:
General rule of good coding is if your going to use the parameter hints you should for all. I know that here with the innerException it looks redundent.

```suggestion
            : base(
                message: "Transaction with the same id already exists.",
                innerException: innerException)
```

Reply:
do the same for all the rest.

Reply:
Resolved.